sdk/worker/servicebus.go

```diff
@@ -51,13 +55,13 @@ type (
 )
 
 // NewServiceBusReceiver is a constructor for the type `ServiceBusReceiver`.
-func NewServiceBusReceiver(connstr string, capacity int) (*ServiceBusReceiver, error) {
+func NewServiceBusReceiver(connstr string, queues ...string) (*ServiceBusReceiver, error) {
```

Comment:
This might be a good place to introduce variadic configuration. I would imagine you'd be adding a backoff policy at some point.

Reply:
or an options struct

Reply:
True! I wondered about doing it, but was hesitant about adding that pattern in yet because it'll be inconsistent with other parts of this code base. I'd be tempted to do a broader sweep of this whole SDK and move all of my constructors over to the updated pattern in a later PR.